extensions/terminal-suggest/src/test/terminalSuggestMain.test.ts

```diff
@@ -56,7 +56,6 @@ function createCodeTestSpecs(executable: string): ITestSpec2[] {
 		{ input: `${executable} |`, expectedCompletions: codeOptions },
 		{ input: `${executable} --locale |`, expectedCompletions: localeOptions },
 		{ input: `${executable} --diff |`, expectedResourceRequests: { type: 'files', cwd: testCwd } },
-		{ input: `${executable} -di|`, expectedCompletions: codeOptions.filter(o => o.startsWith('di')), expectedResourceRequests: { type: 'both', cwd: testCwd } },
```

Comment:
Shouldn't this still be passing? `-di` should fuzzy match `--diff`

Reply:
Yeah and it does. For some reason, the test always passes locally and not in CI.

Reply:
<img width="566" alt="Screenshot 2024-12-16 at 1 13 51 PM" src="https://github.com/user-attachments/assets/98cd2b26-7eac-4ad3-ba2e-43ce46baef48" />